WORKSPACE

```diff
@@ -0,0 +1,101 @@
+workspace(name = "gaia")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_foreign_cc",
+    sha256 = "353e41e99f93c0219994a7c4402a80cd04bd044703818d199c66ff82ec4ee85b",
+    strip_prefix = "rules_foreign_cc-master",
+    url = "https://github.com/bazelbuild/rules_foreign_cc/archive/master.zip",
+)
+
+load("@rules_foreign_cc//:workspace_definitions.bzl", "rules_foreign_cc_dependencies")
+
+rules_foreign_cc_dependencies()
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository", "new_git_repository")
+
+git_repository(
+    name = "com_github_gflags_gflags",
+    remote = "https://github.com/gflags/gflags.git",
+    tag = "v2.2.2",
+)
+
+git_repository(
+    name = "com_github_glog",
+    branch = "Prod",
+    remote = "https://github.com/romange/glog.git",
+)
+
+new_git_repository(
+    name = "pmr",
+    build_file = "cmake.BUILD",
+    commit = "6ca661f421840a7117394c25099f615630b39a04",
+    remote = "https://github.com/romange/pmr.git",
+)
+
+new_git_repository(
+    name = "xxhash",
+    build_file = "xxhash.BUILD",
+    remote = "https://github.com/Cyan4973/xxHash.git",
+    tag = "v0.7.0",
+)
+
+git_repository(
```

Comment:
that's actually is submodule in gaia

Reply:
In other words gaia build treats it as if it's native. At least with CMake because this is how they built it in CMake

Reply:
Bazel doesn't have this feature (since it's a different workspace), so need to include it as a separate repo.
Changing to `local_repository`.